experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
+RUN mkdir /node 
+RUN date
+WORKDIR /node
+RUN mkdir Jmeter node_undertest node_baseline
+RUN git clone http://github.com/gareth-ellis/benchmarking 
+RUN bash /node/benchmarking/experimental/benchmarks/acmeair/setupJmeter.sh
+RUN numactl --hardware
```

Comment:
Ah, I think I may understand what is going on here. Does this output the specs of the machine running the tests?

If so, this should be moved into `run_acmeair_docker.sh`. There is no guarantee that the machine running this layer will be the same machine running the run script. Also, Docker will cache this layer and it will not be run every invocation of `docker build`.